tests/composite_pk/test_create.py

```diff
@@ -0,0 +1,214 @@
+import unittest
+
+from django.db import connection
+from django.test import TestCase
+from django.test.utils import CaptureQueriesContext
+
+from .models import Tenant, User
+
+
+class CompositePKCreateTests(TestCase):
+    """
+    Test the .create(), .save(), .bulk_create(), .get_or_create(), .update_or_create()
+    methods of composite_pk models.
+    """
+
+    maxDiff = None
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.tenant = Tenant.objects.create()
+
+    @unittest.skipUnless(connection.vendor == "sqlite", "SQLite specific test")
```

Comment:
The only reason for this was because they generated different SQLs, but we shouldn't test the SQLs according to @charettes , so this can be simplified.

Reply:
I removed db-specific tests from almost everywhere.